spec/cli_spec.rb

```diff
@@ -73,6 +73,80 @@
 - + - + -
 6 | 7 | 8
 
+GAMEOUTPUT
+
+    end
+  end
+
+  context "a game that ends in a draw" do
+    let(:input) { StringIO.new("0\n1\n4\n2\n5\n3\n6\n8\n7") }
+    let(:output) { StringIO.new }
```

Comment:
This output is used in multiple places, you might as well extract it to the very top level of tests in this file as it's not particularly relevant to the subject matter of the tests, only the mechanisms by which we interact with the code.